CardParts/src/Extensions/UIColor.swift

```diff
@@ -46,4 +46,11 @@ extension UIColor {
     public static var limeGreen: UIColor {get{return UIColor.colorFromHex(0x53B700)}}
     public static var yellowSea: UIColor {get{return UIColor.colorFromHex(0xFFAD00)}}
     public static var superNova: UIColor {get{return UIColor.colorFromHex(0xFFCA00)}}
+    
+    //progressBar View band colors
+    public static var sunflowerYellow: UIColor { get { return UIColor.colorFromHex(0xFFDC00)}}
```

Comment:
Do we need these colors in this file?  I actually would like to start removing this file from CardParts as it pertains a lot to Intuit applications; opposed to making it generic for all applications.

Reply:
I will remove and use generic ones may be  for time being.